### PR TITLE
enable json_to_metadata filter

### PR DESCRIPTION
**Pull request by freedomljc**
Similar to https://github.com/istio/proxy/pull/5144, would like to enable json_to_metadata filter

### CODE REVIEW

File: bazel/extension_config/extensions_build_config.bzl [hunk 1]
```diff
@@ -129,6 +129,7 @@ ENVOY_EXTENSIONS = {
     "envoy.filters.http.header_mutation":               "//source/extensions/filters/http/header_mutation:config",
     "envoy.filters.http.health_check":                  "//source/extensions/filters/http/health_check:config",
     "envoy.filters.http.ip_tagging":                    "//source/extensions/filters/http/ip_tagging:config",
+    "envoy.filters.http.json_to_metadata":              "//source/extensions/filters/http/json_to_metaddata:config",
```
Comment (kyessenov) [inline bazel/extension_config/extensions_build_config.bzl:132]: metaddata -> metadata

**Comment by zirain** [comment]
you also need remove https://github.com/istio/proxy/blob/c5e4f0de4c79ab2979675d613af85b0c7b8c5042/tools/extension-check/wellknown-extensions#L7

File: bazel/extension_config/extensions_build_config.bzl (hunk 1 above)
Comment (freedomljc) [reply bazel/extension_config/extensions_build_config.bzl]: updated, thanks.

**Comment by freedomljc** [comment]
> you also need remove
> 
> https://github.com/istio/proxy/blob/c5e4f0de4c79ab2979675d613af85b0c7b8c5042/tools/extension-check/wellknown-extensions#L7

I think it has been already removed in the first commit.

**Comment by zirain** [comment]
> > you also need remove
> > https://github.com/istio/proxy/blob/c5e4f0de4c79ab2979675d613af85b0c7b8c5042/tools/extension-check/wellknown-extensions#L7
> 
> I think it has been already removed in the first commit.

ok, my bad, I should not review on phone.

**Review by zirain — APPROVED** [review]

**Pull request merged**